--- maildrop-2.9.3/libs/rfc822/Makefile.am.orig	2018-03-23 21:08:13.147577895 +0100
+++ maildrop-2.9.3/libs/rfc822/Makefile.am	2018-03-23 21:13:22.710907697 +0100
@@ -10,7 +10,7 @@
                   rfc822_mkdate.c rfc822_parsedt.c rfc2047u.c \
                   rfc2047.c rfc2047.h imapsubj.c imaprefs.h imaprefs.c \
 		  encodeautodetect.c
-librfc822_la_LIBADD = $(LIBIDN_LIBS)
+librfc822_la_LIBADD = $(LIBIDN_LIBS) -lcourier-unicode
 
 DISTCLEANFILES=rfc822.config
 
--- maildrop-2.9.3/Makefile.am.orig	2018-03-23 21:15:34.417572856 +0100
+++ maildrop-2.9.3/Makefile.am	2018-03-23 21:24:34.564233357 +0100
@@ -102,7 +102,7 @@
 librfc2045_la_SOURCES=
 
 librfc822_la_LIBADD=libs/rfc822/librfc822.la libs/rfc822/libencode.la
-librfc2045_la_LIBADD=libs/rfc2045/librfc2045.la
+librfc2045_la_LIBADD=libs/rfc2045/librfc2045.la libs/numlib/libnumlib.la librfc822.la
 librfc822_la_LDFLAGS=-rpath $(libdir)
 librfc2045_la_LDFLAGS=-rpath $(libdir)
 
