diff -Nur old/maildrop/Makefile.am new/maildrop/Makefile.am
--- old/maildrop/Makefile.am	2004-10-31 03:12:16.000000000 +0000
+++ new/maildrop/Makefile.am	2005-04-09 22:35:14.000000000 +0000
@@ -67,9 +67,8 @@
 mailbot_SOURCES=mailbot.c
 mailbot_DEPENDENCIES=../rfc822/librfc822.la ../liblock/liblock.la \
 	../numlib/libnumlib.la $(DBLIB)
-mailbot_LDADD=../rfc822/librfc822.la ../liblock/liblock.la \
-	../numlib/libnumlib.la $(DBLIB) $(LIBGDBM)
-mailbot_LDFLAGS = -static
+mailbot_LDADD=../rfc822/.libs/librfc822.a ../liblock/.libs/liblock.a \
+	../numlib/.libs/libnumlib.a $(DBLIB) $(LIBGDBM)
 
 EXTRA_DIST = maildroptips.html $(BUILT_SOURCES)\
 	maildrop.lsm testsuite.txt
